tools/configure/cmdArgs.ml

```diff
@@ -182,7 +184,8 @@ let args_options = Arg.align [
     " URL of the coq website";
   "-warn-error", arg_bool (fun p warn_error -> { p with warn_error }),
     "(yes|no) Make OCaml warnings into errors (default no)";
-  "-profile", arg_profile, Profiles.doc
+  "-profile", arg_profile, Profiles.doc;
+  "-debug", arg_set (fun p -> { p with debug = true }), "enable debug information for package detection"
```

Comment:
```suggestion
  "-debug", arg_set (fun p -> { p with debug = true }), "Enable debug information for package detection"
```

Reply:
Waiting for this (or just remove the code, I don't think it's actually useful)